roles/matrix-base/templates/matrix-ip6tables.service.j2

```diff
@@ -0,0 +1,15 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix ip6tables rule to enable IPv6 internet access from containers
+DefaultDependencies=no
+
+[Service]
+Type=oneshot
+Environment="HOME={{ matrix_systemd_unit_home_path }}"
+
+ExecStart=ip6tables -t nat -A POSTROUTING -s fd00::/80 ! -o docker0 -j MASQUERADE
```

Comment:
```suggestion
ExecStart={{ matrix_host_command_sh }} -c "ip6tables -t nat -A POSTROUTING -s fd00::/80 ! -o docker0 -j MASQUERADE"
```

Changing this made it work for me. I don't know if this is the right solution.

Reply:
That'd be unnecessary shell invocation. It works, but it's not the best. Something involving an absolute path to `env` is probably the right solution. And I also forgot the second import `ip6tables` rule in there.

Like I said, this needs more testing before it's ready :p

Reply:
It is recommended to put absolute paths into ExecStart=, in this case that would be `/usr/sbin/ip6tables`